### ISSUE TITLE
selecting an autosuggested topic should behave consistently

### ISSUE THREAD

**Issue by hancush** [datamade/la-metro-councilmatic#454]
<blockquote>
There seems to be some difference in behavior when a selection is made based on whether there are matching results (sometimes?).  Shelly and I noticed that if there is some behavioral differences when selecting suggestions.  Is there a difference in behavior here? Are matched results from the selection treated differently or is something weird going on here?<br />
Some suggestion to try:<br />
-Partially typed “sa” and select Sabotage. (No search performed/triggered)<br />
-Partially typed “bus” and select Bus 60'Ze W/ Byd-5. (Search performed/triggered)
</blockquote>

**Comment by jmithani** [comment]
[This commit](https://github.com/datamade/la-metro-councilmatic/pull/445/commits/6b1ebee92ee5b8659d5b26e1f828277d79313bde) edits the `jquery.autocomplete.js` library so that when the user clicks on `No topics found` the search bar is cleared and nothing happens. 

For cases like `Sabotage` are trickier. The [`/autocomplete/` endpoint](https://lametro.datamade.us/autocomplete/?query=sa) only returns metadata about the facet, and once the suggestion is clicked it hits the [`/topic/`endpoint](https://lametro.datamade.us/topic/?guid=54a484d7-5390-4701-afe0-e4748fe122bb) which then returns the `status_code` of 404.

Filtering to only show topics with `status_code == 200` is expensive and would slow the suggestions down. Reworking the view would change our set up quite a bit. Since the example of `Sabotage` should be resolved once the topics are imported from the live instance (rather than the test data), I wouldn't recommend we spend time handling this case, which seems very rare.

**Comment by hancush** [comment]
@jmithani thanks – generally, i agree with your assessment!

i do have a question: it is my understanding that the legistar indexes endpoint will contain all topics associated with a concept in ses, however i don't see an index for "Sabotage" at http://webapi.legistar.com/v1/metrotest/indexes/?$filter=IndexName eq 'Sabotage'. that makes me wonder whether the hints endpoint will return all topics in ses, regardless of whether they're associated with anything/synced to legistar. can you follow up with steve (on the feedback thread) asking for clarification?

tl;dr - is it possible for the hints endpoint to suggest topics that have not been synced to legistar?

**Comment by hancush** [comment]
closed in #445!
